CIP-XXXX/README.md

```diff
@@ -0,0 +1,181 @@
+---
+CIP: ?
+Title: Standard JSON encoding for Domain Types
+Category: Tools
+Status: Proposed
+Authors:
+    - Vladimir Kalnitsky <klntsky@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2024-02-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Canonical JSON encoding for Cardano domain types lets the ecosystem converge on a single way of serializing data to JSON, thus freeing the developers from repeating roughly the same, but slightly different encoding/decoding logic over and over.
+
+## Motivation: why is this CIP necessary?
+
+Cardano domain types have canonical CDDL definitions (for every era), but when it comes to use in web apps, where JSON is the universally accepted format, there is no definite standard. This CIP aims to change that.
+
+The full motivation text is provided in [CPS-11](https://github.com/cardano-foundation/CIPs/pull/742).
+
+## Specification
+
+This CIP is expected to contain multiple schema definitions for Cardano Eras starting from Babbage.
```

Comment:
Why dont we allow new schemas to be added to this set via new CIPs? 
in a similar to the approach to CIP30 extensions and the CDDL proposals

Reply:
Because there is no point in keeping multiple CIPs for the same thing for different eras. From the data layout perspective, eras are mostly the same, so the jsons will be almost the same.

Once we "set in stone" a schema for a given era, we are not going to update it, because of course the data layout will stay the same during an era.

In other words, making a CIP extension is pointless because there is nothing to extend, only additions are possible.